plugins/http/src/commands.rs

```diff
@@ -386,13 +387,34 @@ pub async fn fetch_send<R: Runtime>(
 pub(crate) async fn fetch_read_body<R: Runtime>(
     webview: Webview<R>,
     rid: ResourceId,
-) -> crate::Result<tauri::ipc::Response> {
+    channel: tauri::ipc::Channel<&[u8]>,
+) -> crate::Result<()> {
     let res = {
         let mut resources_table = webview.resources_table();
         resources_table.take::<ReqwestResponse>(rid)?
     };
+
     let res = Arc::into_inner(res).unwrap().0;
-    Ok(tauri::ipc::Response::new(res.bytes().await?.to_vec()))
+    let mut stream = res.bytes_stream();
```

Comment:
can't we do the same streaming behavior using `Response::chunk` method, instead of pulling another crate to iterate over the stream?